tests/unit/connection_parameters_tests.py

```diff
@@ -276,11 +276,17 @@ def test_port(self):
         params.port = 5672
         self.assertEqual(params.port, 5672)
 
+        params.port = '5672'
+        self.assertEqual(params.port, 5672)
+
+        params.port = 1.5
+        self.assertEqual(params.port, 1)
```

Comment:
This is a change from current behavior, though I'm not all that concerned about it. @michaelklishin - thoughts?